spec/spec_helper.rb

```diff
@@ -120,7 +120,8 @@
 
   c.allow_http_connections_when_no_cassette = false
   c.default_cassette_options = {
-    :allow_unused_http_interactions => false
+    :allow_unused_http_interactions => false,
+    :decode_compressed_response     => true
```

Comment:
OK, I will remove this change from this PR. I spend sometime for looking for this option and wanted other people to not waste time.